docs/AppendixA-FAQ.md

```diff
@@ -1 +1,151 @@
 # Appendix A - FAQ
+
```

Comment:
I think this whole section needs to be removed or re-organized. It includes many different levels of troubleshooting, but also how to run and re-run scripts. 
I would remove A.1 and the first sub-header of A.2 since they should be in the FAQ table
I would then move the rest of anything in section A.2 to a sub-section of 4.0-Scripts. We can have there section 1; just how the perl scripts are documented, and section 2, the text from here where we cna label it as common re-run of some scripts.

Reply:
I will do this in this pull request here: https://github.com/aces/Loris-MRI/pull/230
(which is based off of yours here).

Reply:
@MounaSafiHarab Does that mean I don't touch this and you will do it later? Just making sure

Reply:
I think it might be best this way as I am based off of your pull request. I will re-read your sections, merge it, then re-base mine, making my task there a lot easier, and your reviewing a lot faster as you will be able to track changes more easily:)